Source/MQTTnet/Client/MqttClient.cs

```diff
@@ -500,7 +499,7 @@ private void StartReceivingPackets(CancellationToken cancellationToken)
             _packetReceiverTask = Task.Factory.StartNew(
                 () => ReceivePacketsAsync(cancellationToken),
                 cancellationToken,
-                TaskCreationOptions.LongRunning, 
+                TaskCreationOptions.LongRunning,
```

Comment:
TaskCreationOptions.LongRunning is pretty useless, just use `Task.Run` as `LongRunning` forces the creation of a thread which isn't specifically for all processing.

See http://blog.i3arnon.com/2015/07/02/task-run-long-running/ for more info.